pkg/pillar/cmd/zedrouter/flowstats.go

```diff
@@ -327,31 +351,33 @@ func flowMergeProcess(entry *netlink.ConntrackFlow, instData networkAttrs) flowS
 	ipFlow.aclNum = entry.Mark & markMask
 	ipFlow.appNum = uint8(entry.Mark >> appShiftBits)
 	AppNum = int(ipFlow.appNum)
-	if AppNum == 0 { // only handle App related flow stats
+	if AppNum == 0 || AppNum == 255 { // only handle App related flow stats
```

Comment:
Are we reserving 255 somewhere?

Reply:
ok, I'll remove this check. I'll ask Gopi to set the non-App ACL always to zero, otherwise it does not break anything, just waste some processing in this function.